automation/src/test/resources/reference.conf

```diff
@@ -8,3 +8,6 @@ leonardo.currentWelderImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:ed93
 leonardo.bioconductorImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.9"
 leonardo.rstudioBaseImageUrl = "us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.2"
 leonardo.rstudioBioconductorImageUrl = "us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.3"
+leonardo.pubsubGoogleProject = "broad-dsde-dev"
```

Comment:
Does this work for tests run in broad-dsde-qa? I wonder if these pubsub configs should live here: https://github.com/broadinstitute/firecloud-automated-testing/blob/master/configs/leonardo/application.conf.ctmpl

Reply:
currently the pubsub topic is only created in broad-dsde-dev via terraform. All other environments need to be run with tf change as well... @jdcanas I think we can do that now since things are working now?

Reply:
Might be worth jumping on a quick call for this. I'm working on the terraform subtask. As I understand it though, the tests in qa run in a fiab, which will get their topic name from https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/leonardo/leonardo.conf.ctmpl#L88 , and thus the only thing that needs to be done is add the appropriate role to the qa svc account (which we've done). If the service account has the appropriate permissions, our google2 pubsub impl automatically creates whatever topic you tell it to use. 

I do think I need to move leonardo.pubsubGoogleProject to the config you mention though rob, based on this understanding. However, it should be sufficient to keep a static project here, as the auto test code only write to pubsub, never read to avoid conflicts. The topic in leonardo (which is randomly generated from the linked config above) is the one that goes full cycle and is unique.

Reply:
My point was that the pubSub project might be `broad-dsde-dev` or `broad-dsde-qa` for automation tests -- so the project configured here needs to match the project configured in Leo. So I don't think we can have a static project in this automation/reference.conf, we should template it in https://github.com/broadinstitute/firecloud-automated-testing/blob/master/configs/leonardo/application.conf.ctmpl.

So the google2 impl creates the topic if it doesn't exist? Do we need to terraform it then?

Reply:
Happy to jump on a call too, i could be misunderstanding something.

Reply:
> My point was that the pubSub project might be broad-dsde-dev or broad-dsde-qa for automation tests -- so the project configured here needs to match the project configured in Leo. 

Agree...

> So the google2 impl creates the topic if it doesn't exist? Do we need to terraform it then?
I think terraform change is still needed since it sets up some permission https://github.com/broadinstitute/terraform-terra/pull/111/files#diff-145fc03d6d7602a93c883b635fb3b01eR9